app/src/main/java/com/eveningoutpost/dexdrip/Services/WixelReader.java

```diff
@@ -332,7 +408,13 @@ static Long timeForNextRead() {
             // A new packet should arrive but chance is we have missed it...
             return 30000L;
         }
-        return (DEXCOM_PERIOD - gapTime) + 2000;
+
```

Comment:
Are you sure that this is what you want to do?
If you have both local receivers and an http client, you can go to the local clients and get the information  from there.
This seems to me to be right only when http client is the only one around.